apis/python/src/tiledbvcf/dataset.py

```diff
@@ -351,17 +350,12 @@ def ingest_samples(
         :param list of str sample_uris: CSV list of sample names to include in
             the count.
         :param int threads: Set the number of threads used for ingestion.
-        :param int thread_task_size: Set the max length (# columns) of an
-            ingestion task. Affects load balancing of ingestion work across
-            threads, and total memory consumption.
-        :param int memory_budget_mb: Set the max size (MB) of TileDB buffers before flushing
-            (default = 1024).
+        :param int memory_budget_mb: The total memory budget for ingestion (MiB)
+        :param float memory_budget_ratio: Ratio of total system memory used for ingestion (overrides 'memory_budget_mb')
```

Comment:
I think we're missing the C API and py bindings for this?

Reply:
Can we spell out what this is a ratio of?

Reply:
Good catch, the C API and python bindings are not done yet. Will add those.